files/ru/web/api/htmlcanvaselement/webglcontextlost_event/index.html

```diff
@@ -0,0 +1,78 @@
+---
+title: 'HTMLCanvasElement: событие webglcontextlost'
+slug: Web/API/HTMLCanvasElement/webglcontextlost_event
+translation_of: Web/API/HTMLCanvasElement/webglcontextlost_event
+tags:
+- Event
+- Reference
+- WebGL
+---
+<div>{{APIRef}}</div>
+
+<p>Событие <code><strong>webglcontextlost</strong></code> <a href="/ru/docs/Web/API/WebGL_API">WebGL API</a> вызывается когда {{Glossary("User agent")}}
+    обнаруживает, что буфер, связанный с {{domxref("WebGLRenderingContext")}}, потерян.</p>
+
+<table class="properties">
+    <tbody>
+    <tr>
+        <th scope="row">Всплытие</th>
+        <td>Да</td>
+    </tr>
+    <tr>
+        <th scope="row">Отменяемое</th>
+        <td>Да</td>
+    </tr>
+    <tr>
+        <th scope="row">Интерфейс</th>
+        <td>{{domxref("WebGLContextEvent")}}</td>
+    </tr>
+    <tr>
+        <th scope="row">Свойство обработчика событий</th>
```

Comment:
Думаю, что так будет проще для понимания, так как речь идёт об `onchange`, `onclick` и т.п. Если в этом пункте установлен `none`, значит такое свойство не существует.
```suggestion
        <th scope="row">Свойство для обработки события</th>
```

Reply:
@myshov тогда возможно лучше "none" перевести как "отсутствует"?

Reply:
можно, но и "нет" тоже вроде ок